norddrop/src/device/mod.rs

```diff
@@ -289,19 +283,17 @@ impl NordDropFFI {
         }
 
         let result = self.rt.block_on(async {
-            self.instance
+            let transfers = self
+                .instance
                 .lock()
                 .await
                 .as_mut()
                 .ok_or(ffi::types::NORDDROP_RES_NOT_STARTED)?
+                .storage()
                 .transfers_since(since_timestamp)
-                .map_err(|err| {
-                    error!(self.logger, "Failed to get transfers: {:?}", err);
-                    match err {
-                        drop_transfer::Error::StorageError => ffi::types::NORDDROP_RES_DB_ERROR,
-                        _ => ffi::types::NORDDROP_RES_DB_ERROR,
-                    }
-                })
+                .await;
+
+            Ok::<Vec<drop_storage::types::Transfer>, ffi::types::norddrop_result>(transfers)
```

Comment:
Doesn't simple `Ok(transfers)` work?

Reply:
Does `Result::Ok(transfers)` work? The `Result` is declared in this file for exactly the `norddrop_result` error type so it should work